manifests/kustomize/options/catalog/sample-catalog.yaml

```
@@ -0,0 +1,201 @@
+    messages = [
+        {"role": "system", "content": "You are a pirate chatbot who always responds in pirate speak!"},
+        {"role": "user", "content": "Who are you?"},
+    ]
+    chatbot = pipeline("text-generation", model="mistralai/Mistral-7B-Instruct-v0.3")
+    chatbot(messages)
+    ```
+
+
+    ## Function calling with `transformers`
+
+    To use this example, you'll need `transformers` version 4.42.0 or higher. Please see the 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)